src/main/webapp/profileScript.js

```diff
@@ -146,3 +147,30 @@ function addRoute(newRoute) {
     document.getElementById("future-routes").appendChild(card);
   }
 }
+
+function showSubmitAvatar() {
+  document.getElementById("submit-avatar").classList.toggle("show");
+}
+
+function showAvatar() {
+  fetch("/profile-image", {
+    method: "GET",
+  })
+    .then((response) => response.json())
+    .then((avatarName) => {
+      try {
+        document.getElementById("avatar-image").innerHTML =
+          '<img src="https://storage.cloud.google.com/user-image-globes/' +
+          avatarName +
+          '"' +
+          'alt="Profile picture"/>';
+      } catch (exception) {
```

Comment:
I am not sure what exception might happen here, but you could probably remove the try/catch.